File: Android/src/co/hexi/TestGame/TestGameAndroidStarter.scala
package co.hexi.TestGame

import com.badlogic.gdx.backends.android.{AndroidApplicationConfiguration, AndroidApplication}
import android.os.Bundle
import hexico.testgame.TestGame

class TestGameAndroidStarter extends AndroidApplication {
  override def onCreate(savedInstanceState: Bundle) {
    super.onCreate(savedInstanceState)
    val cfg = new AndroidApplicationConfiguration
    cfg.useAccelerometer = false
    cfg.useCompass = false
    cfg.useWakelock = false
    cfg.useGL20 = true
    initialize(new TestGame, cfg)
  }
}

File: Android/src/co/hexi/TestGame/TestGameAndroidStarter.java
package co.hexi.TestGame;

import android.app.Activity;
import android.os.Bundle;
import com.badlogic.gdx.backends.android.AndroidApplication;
import com.badlogic.gdx.backends.android.AndroidApplicationConfiguration;
import hexico.testgame.TestGame;

/**
 * Created with IntelliJ IDEA.
 * User: Alan
 * Date: 24/04/13
 * Time: 20:02
 * To change this template use File | Settings | File Templates.
 */
public class TestGameAndroidStarter extends AndroidApplication {
    @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        AndroidApplicationConfiguration cfg = new AndroidApplicationConfiguration();
        cfg.useAccelerometer = false;
        cfg.useCompass = false;
        cfg.useWakelock = false;
        cfg.useGL20 = true;
        initialize(new TestGame(), cfg);
    }
}